.circleci/deployment/commands.yml

```diff
@@ -1,4 +1,30 @@
 # commands:
+  init-deploy:
+    steps:
+      - checkout
+      - sudo-check
+      - cf-check
+
+  build-and-tag-images:
+    parameters:
+      backend-appname:
+        default: tdp-backend
+        type: string
+      frontend-appname:
+        default: tdp-frontend
+        type: string
+    steps:
+      - setup_remote_docker
+      - run:
+          name: Tag images
```

Comment:
I see that we're just tagging/uploading. Beyond MVP, how should we prevent duplication of our [rafttech hub](https://hub.docker.com/repository/docker/rafttech/tdp-backend/general) like is used by [clamav here](https://github.com/raft-tech/TANF-app/blob/develop/tdrs-backend/manifest.clamav.yml#L8)?

Reply:
Im not sure what you're asking. The existing raftech dependency images can stay in that repo or we can move them to Nexus. They shouldnt change in our raftech repo so there isn't a dire need to move them over unless we just want everything in one place. As for the rest of the dependencies, those arent being re-tagged/uploaded but they do exist in Nexus and are being pulled from it.